packages/app/source/server/publications/publications.js

```diff
@@ -1,7 +1,7 @@
 Space.messaging.Publication.extend(Todos, 'TodosPublication', {
```

Comment:
Let's move this up to the /server directory. It's likely this will become a pattern as most apps should just have a single Publication class